### PR TITLE
Feature/profile pictures

### PR DESCRIPTION
**Pull request by WalterRestifo**
Hello, my App should become a matching app for volleyball players. I have a databank with some players and you can filter it, so that only the players with the desired criteria are showed. You can make an own profile in the profile section to get added to the list of the players. In this PR I added a new page for uploading an own profile picture. This profile picture is showed in the list of all players on the main page. If you don't choose any picture, a default one is chosen for you

### CODE REVIEW

File: cypress/support/commands.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+// declare global {
+//   namespace Cypress {
+//     interface Chainable {
+//       login(email: string, password: string): Chainable<void>
+//       drag(subject: string, options?: Partial<TypeOptions>): Chainable<Element>
+//       dismiss(subject: string, options?: Partial<TypeOptions>): Chainable<Element>
+//       visit(originalFn: CommandOriginalFn, url: string, options: Partial<VisitOptions>): Chainable<Element>
+//     }
+//   }
+// }
```
Comment (MerleMak) [inline cypress/support/commands.ts:38]: please don't commit code which is commented out, only code you are actually using :)

**Review by MerleMak — COMMENTED** [review]
Hi Walter, I left some minor remarks for you. 
Mainly concerning comments :D Please make sure to only commit things you actually want to get reviewed, so no code thats commented out, also for your reviewer to save some time :)
Good work otherwise. I am curious for your test to be completed!
Keep it up! 🙌🏽

File: cypress/e2e/spec.cy.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+});
+
+describe("My First Test", () => {
+  it("Does not do much!", () => {
+    expect(true).to.equal(true);
+  });
+});
+
+describe("My Second Test", () => {
+  it("Visits the Kitchen Sink and should navigate to a new URL by clicking the type link", () => {
```
[the remark this answers is not in the document]
Comment (WalterRestifo) [reply cypress/e2e/spec.cy.ts:16]: sorry I was playing around with cypress a few days ago and forgot to make a new feature branch 😅

File: cypress/e2e/spec.cy.ts [hunk 3]
```diff
@@ -0,0 +1,25 @@
+export {};
+
+describe("empty spec", () => {
+  it("passes", () => {
+    cy.visit("https://example.cypress.io");
```
[the remark this answers is not in the document]
Comment (WalterRestifo) [reply cypress/e2e/spec.cy.ts:5]: I had to learn cypress because of an email of neuefische about a test and wanted to play a little with it 🤷🏽

**Pull request merged**